app/templates/course/stage/code-examples.hbs

```diff
@@ -58,7 +58,12 @@
       </div>
     {{else if (gt this.sortedSolutions.length 0)}}
       {{! @glint-expect-error: not ts-ified yet }}
```

Comment:
This is a .ts component now, looks like the comment above is hiding another error. Can we look into what it is?